test/mainnet-fork/SpkFarm.t.sol

```diff
@@ -160,13 +161,18 @@ contract MainnetControllerSPKFarmWithdrawSuccessTests is MainnetControllerSPKFar
         assertEq(usds.balanceOf(address(almProxy)),                  0);
         assertEq(ISPKFarmLike(spkFarm).balanceOf(address(almProxy)), 1_000_000e18);
 
+        skip(1 days);
+
+        uint256 rewardTokenEarned = ISPKFarmLike(spkFarm).earned(address(almProxy));
+
         vm.prank(relayer);
         mainnetController.withdrawUSDSFromSPKFarm(spkFarm, 1_000_000e18);
 
         assertEq(rateLimits.getCurrentRateLimit(withdrawKey), 9_000_000e18);
 
         assertEq(usds.balanceOf(address(almProxy)),                  1_000_000e18);
         assertEq(ISPKFarmLike(spkFarm).balanceOf(address(almProxy)), 0);
+        assertEq(IERC20(Ethereum.SPK).balanceOf(address(almProxy)),  rewardTokenEarned);
```

Comment:
This is forked at a constant block so we should assert the value. Also as a general note, caching values and asserting like this can actually be somewhat dangerous as they can both be zero and it can pass without us knowing that something is wrong. At the minimum we should check its above zero, but better practice is to check the value itself when at a forked block.

Reply:
makes sense, updated